src/renderer/initializers/catalog.tsx

```diff
@@ -64,7 +44,29 @@ async function onClusterDelete(clusterId: string) {
   DeleteClusterDialog.open({ cluster, config });
 }
 
-export function initCatalog() {
-  initWebLinks();
-  initKubernetesClusters();
+interface Dependencies {
+  openCommandDialog: (component: React.ReactElement) => void;
+}
+
+export function initCatalog({ openCommandDialog }: Dependencies) {
```

Comment:
In future, we should move calls of `init` functions to `instantiate`.  In that way, we don't need to worry about timing.